angular/src/menu.html

```diff
@@ -0,0 +1,214 @@
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/1/"><div>Product Name Here</div></li>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/4/"><div>Product Name Here</div></li>
+                        </ul>
+                   </div>
+                   <div class="row">     
+                        <ul>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/4/"><div>Product Name Here</div></li>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/2/"><div>Product Name Here</div></li>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/1/"><div>Product Name Here</div></li>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/3/"><div>Product Name Here</div></li>
+                        </ul>
+                   </div>
+               </div> <!-- END TAB 2-->
+               
+               <div ng-show="tab.isSet(3)">
+                   <div class="row">     
+                        <ul>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/4/"><div>Product Name Here</div></li>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/1/"><div>Product Name Here</div></li>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/2/"><div>Product Name Here</div></li>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/3/"><div>Product Name Here</div></li>
+                        </ul>
+                   </div>
+                   <div class="row">     
+                        <ul>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/3/"><div>Product Name Here</div></li>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/4/"><div>Product Name Here</div></li>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/2/"><div>Product Name Here</div></li>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/1/"><div>Product Name Here</div></li>
+                        </ul>
+                   </div>
+               </div> <!-- END TAB 3 -->
+               
+               <div ng-show="tab.isSet(4)">
+                   <div class="row">     
+                        <ul>
+                            <li class="col-xs-12 col-sm-3"><img src="http://lorempixel.com/150/200/nature/3/"><div>Product Name Here</div></li>
```

Comment:
Remember, `col-xs-12` is the _default_ for size `xs`... Save yourself the typing!

While you're at it, save yourself even _more_ typing by using `ng-repeat` to duplicate those `<li>`'s... like we did with Vue JS.